chat-sdk-core/src/main/java/co/chatsdk/core/handlers/ThreadHandler.java

```diff
@@ -79,4 +79,11 @@
 
     Completable pushThread(Thread thread);
     Completable pushThreadMeta(Thread thread);
+
+    Observable<Thread> on(Thread thread);
+    void off(Thread thread);
+
+    Observable<Thread> metaOn(Thread thread);
+    void metaOff(Thread thread);
+
```

Comment:
Why are these methods needed? We try to keep the number of methods in the handler to an absolute minimum because it's supposed to be as generic as possible.